src/formatters/html.js

```diff
@@ -20,6 +20,10 @@ function htmlEscape(text) {
   return html;
 }
 
+HtmlFormatter.prototype.prepareContext = function(context) {
+  BaseFormatter.prototype.prepareContext.call(this, context);
+};
```

Comment:
ok, np, but you can try this at http://benjamine.github.io/jsondiffpatch/demo/index.html on the browser console:
``` js
var htmlFormatter = new jsondiffpatch.formatters.html.HtmlFormatter();
var context = {};
htmlFormatter.prepareContext(context);
console.log(context);
```
you'll see context is a properly initialized object, instead of getting something like `prepareContext is undefined`.
it's ok though np, thanks!